adapters/emtv/emtv.go

```diff
@@ -80,6 +81,11 @@ func (a *adapter) MakeRequests(request *openrtb2.BidRequest, reqInfo *adapters.E
 			adapterRequests = append(adapterRequests, adapterReq)
 		}
 	}
+
+	if len(adapterRequests) == 0 {
+		return nil, []error{errors.New("found no valid impressions")}
```

Comment:
Could you please add test case for this change? @EngageMediaHB

Reply:
@gargcreation1992 added